app/views/pages/about.html.erb

```diff
@@ -0,0 +1,22 @@
+<div class="container release-notes col-md-12">
+  <h1 class="page-heading">About the New Catalog</h1>
+
+  <p>Initially released on 28 April 2016, this catalog is the result of a nearly two year long project to replace our twenty year old <%= t('blacklight.voyager') %> and incorporate the best features of new types of library search technology.</p>
```

Comment:
Similarly here:

<p>Initially released on 28 April 2016, this catalog is the result of an almost two-year-long project to replace our 20-year-old <%= t('blacklight.voyager') %> and incorporate the best features of new types of library search technology.</p>